addons/surgery/functions/fnc_hasTourniquetAppliedToAdditional.sqf

```diff
@@ -0,0 +1,23 @@
+#include "..\script_component.hpp"
+/*
+ * Author: Glowbal
+ * Checks if the unit has a tourniquet applied on the specified body part.
+ *
+ * Arguments:
+ * 0: Unit <OBJECT>
+ * 1: Body Part <STRING>
+ *
+ * Return Value:
+ * Has Tourniquet Been Applied <BOOL>
+ *
+ * Example:
+ * [player, "leftleg"] call ace_medical_treatment_fnc_hasTourniquetAppliedTo
```

Comment:
wrong example, also function name should be "hasAdditionalTourniquetAppliedTo" as its more clear imo